homeassistant/components/osc/services.yaml

```diff
@@ -0,0 +1,25 @@
+osc_send:
+  name: Send OSC message
+  description: Send an OSC message to a specific entity
+  fields:
+    entity_id:
+      description: The entity_id of the OSC device
+      example: "sensor.my_osc_device"
+      selector:
+        entity:
+          integration: osc
```

Comment:
this is good as it makes the dev tools/automation UI do the right thing. When I mentioned above to use the selector, do it in your code too so the backend does the same validation